hack/verify-util-pkg.sh

```diff
@@ -0,0 +1,48 @@
+#!/bin/bash
+
+# Copyright 2017 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+KUBE_ROOT=$(dirname "${BASH_SOURCE}")/..
+
+cd "${KUBE_ROOT}"/pkg/util
```

Comment:
Rather than `cd` into the directory, I'd rather refer to the directory absolutely as `"${KUBE_ROOT}/pkg/util"` or use pushd/popd. Plenty of examples of this in `hack/` (e.g. https://github.com/kubernetes/kubernetes/blob/master/hack/verify-godeps.sh#L88)